sycl/doc/design/Fuzzing.md

```diff
@@ -0,0 +1,104 @@
+# Fuzzing plan for intel/llvm
+
+The main product which is being developed at intel/llvm repo is a SYCL
+implementation. At high-level, it consists of two compiler and runtime and
```

Comment:
```suggestion
implementation. At high-level it consists of two compilers and a runtime,
```

Reply:
I think Alexey meant to say "two components: a compiler and a runtime" rather than "two compilers and a runtime".